scripts/utils.js

```diff
@@ -170,7 +170,8 @@ const sortAlphabetically = (object) => {
  * @returns {IconData[]} The sorted icons data.
  */
 export const formatIconData = (iconsData) => {
-	const icons = iconsData.map((icon) => {
+	const iconsDataCopy = structuredClone(iconsData);
```

Comment:
Shouldn't this be?

```suggestion
	const iconsDataCopy = globalThis.structuredClone(iconsData);
```

Reply:
XO didn't report this issue, so it should be fine. And we only used `node` runtime.

But both are OK for me. Feel free to apply your suggestion if you prefer the `globalThis.structuredClone`.